### PR TITLE
Some minor cleanups of ThemeData and focus stuff.

### PR DESCRIPTION
**Pull request by gspencergoog**
## Description

This contains some minor cleanups that I found while working on the focus implementation.

There are some formatting changes, a minor functionality change in FocusNode, and adding focus traversal support to the text form field demo in the flutter gallery.

### CODE REVIEW

File: packages/flutter/lib/src/material/theme_data.dart [hunk 1]
```diff
@@ -732,6 +729,9 @@ class ThemeData extends Diagnosticable {
   /// A theme for customizing the color, elevation, and shape of a bottom sheet.
   final BottomSheetThemeData bottomSheetTheme;
 
+  /// A theme for customizing colors, shape, elevation, and behavior of a [SnackBar].
```
Comment (darrenaustin) [inline packages/flutter/lib/src/material/theme_data.dart]: Not a big deal, but why move this?

File: packages/flutter/lib/src/material/theme_data.dart [hunk 2]
```diff
@@ -992,61 +992,61 @@ class ThemeData extends Diagnosticable {
...
-           (otherData.bottomAppBarTheme == bottomAppBarTheme) &&
-           (otherData.colorScheme == colorScheme) &&
-           (otherData.dialogTheme == dialogTheme) &&
-           (otherData.floatingActionButtonTheme == floatingActionButtonTheme) &&
-           (otherData.typography == typography) &&
-           (otherData.cupertinoOverrideTheme == cupertinoOverrideTheme) &&
-           (otherData.snackBarTheme == snackBarTheme) &&
-           (otherData.bottomSheetTheme == bottomSheetTheme);
+    return (otherData.brightness == brightness)
+        && (otherData.primaryColor == primaryColor)
```
Comment (darrenaustin) [inline packages/flutter/lib/src/material/theme_data.dart]: Is this the preferred flutter style for this? It looked like the original and this were both listed as 'good' in the style guide.

**Review by darrenaustin — APPROVED** [review]
LGTM

File: examples/flutter_gallery/lib/demo/material/text_form_field_demo.dart [hunk 3]
```diff
@@ -166,6 +166,26 @@ class TextFormFieldDemoState extends State<TextFormFieldDemo> {
     ) ?? false;
   }
 
+  bool _handleShortcuts(FocusNode node, RawKeyEvent event) {
+    if (event is RawKeyDownEvent) {
+      print('Scope got key event: ${event.logicalKey}, $node');
```
Comment (HansMuller) [inline examples/flutter_gallery/lib/demo/material/text_form_field_demo.dart]: We don't usually leave (console) print statements in the demos. Maybe there's something worth displaying on-screen, or maybe add a comment?

**Comment by gspencergoog** [comment]
I'm just going to abandon this PR.  There's nothing really all that useful left in it: the one code change doesn't add anything (it's the same behavior), and the gallery example change doesn't really work yet, since the TAB gets entered into the field when you hit it, and I don't think we should put non-functional things into the gallery. I'll re-add that once we're done fixing the text field APIs so they can deal with that. The rest are just comment changes I can put into another PR.

**Pull request closed without merging**